src/libsyncengine/syncpal/syncpal.cpp

```diff
@@ -67,68 +66,69 @@ SyncPal::SyncPal(const SyncPath &syncDbPath, const std::string &version, bool ha
     createSharedObjects();
 }
 
-SyncPal::SyncPal(int syncDbId, const std::string &version) : _logger(Log::instance()->getLogger()), _syncDbId(syncDbId) {
+SyncPal::SyncPal(const int syncDbId_, const std::string &version) : _logger(Log::instance()->getLogger()) {
```

Comment:
The '_' at the end of the parameter's name is not the usual naming convention that we use. Is it voluntary?

Reply:
This the way I usually resolve the name conflict between the input parameter and the method name (accessor). 
Would you suggest something else?

Reply:
No, this is fine for me then 👍